AUTHORS.md

```diff
@@ -26,4 +26,4 @@ By adding your name to this document, you agree to release all your contribution
 - [Frederik Arp Frandsen](https://github.com/fredeaf)
 - [Quentin Stiévenart](https://github.com/acieroid)
 - [Stephen Tetley](https://github.com/stephentetley)
-- [Matthew Lutze](https://github.com/mlutze)
```

Comment:
Please don't remove me from this file :)

Reply:
Sorry!

Reply:
Haha! Yes, please keep everyone there :)

Reply:
Don't drink and merge as they say.